test/util/ip.go

```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func GetTopLevelDir() string {
```

Comment:
again I tried it and it didn't work, I don't really understand why because under the hood the split function provided by that module uses string.Split, but ok...